relay-general/src/store/transactions/processor.rs

```diff
@@ -1675,7 +1684,7 @@ mod tests {
         let json = r#"
         {
             "type": "transaction",
-            "transaction": "/foo/2fd4e1c67a2d28fced849ee1bb76e7391b93eb12/user/123/0/",
+            "transaction": "/foo/rule-target/user/123/0/",
```

Comment:
The reason for these changes is to avoid UUID scrubbing rules running first in all fields. They are already running for `123` so that is good enough.